ipa_check_consistency

```diff
@@ -609,6 +611,38 @@ dns_zones() {
   fi
 }
 
+certificates() {
+  local server="" count="" state=""
+  local dir="${TMP_DIR}/certificates"
+
+  mkdir -p "$dir"
+  printf "%-${FIRST_COLUMN}s" "Certificates"
+  for server in "${SERVERS[@]}"; do
+    (
+    if ! count=$(query_ldap "ou=certificateRepository,ou=ca,o=ipaca" \
+      "(certStatus=*)" \
+      "dn" "one"); then
+      count="n/a"
```

Comment:
Ok, I'll query LDAP to see if we are a ca-less install and act accordingly, but it might take some time until I have an updated patch ready.